smoke-tests/src/test/java/com/newrelic/smoke/DaemonAgentSmokeTest.java

```diff
@@ -28,6 +28,7 @@ void test() throws InterruptedException {
     appContainer.start();
 
     assertEdgeHasEventsAndMetrics(60);
+    appContainer.stop();
```

Comment:
This is unnecessary - `cleanupContainer` ensures that the app container is stopped after the test, regardless of whether an exception occurs during the test. 

Same applies to `DaemonAppSmokeTest` and ``JlinkSmokeTest`.

Reply:
Fixed.